data/hai/hai reveal 7 epilogue.txt

```diff
@@ -0,0 +1,820 @@
+		attributes uninhabited
+		landscape land/station0
+		description `There's a partially-dismantled Hai Geocoris with its innards rearranged into a shielded station core here.`
+		security 0.2
+	
+	
+mission "HRF 04 Escort Republic Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Mooncreek"
+	destination "Maelstrom"
+	to offer
+		has "HRF 03 Defend Mooncreek Construction Site: done"
+	on offer
+		event "mooncreek operational"
+		payment 10000000
+		conversation
+			`With a strong force ensuring that no one can interrupt the final construction stage, space-based cranes set about removing the three outer sections of the Geocoris and moving them to the steel ring deployed around the original station core. Within a few hours all seven of the new Geocoris have been dismantled and integrated to the station structure, using lessons learnt from the construction of Mountaintop to rapidly finish a station that is fully battle-ready, if lacking in some of the amenities of life. In a reversal of the Mountaintop priorities, the construction workers unpack and assemble a massive holoprojector while wearing environmental suits before they begin unpacking the contents of the Geocoris storage pods and engaging principle life support systems.`
+			`	As soon as the life support systems around the docking section are engaged, the security ships begin landing those damaged by the hastily assembled pirate response in order to affect critical repairs.`
+			`	As you're attending to your own ships after their extended guard duty, you get pinged by a hail that has been relayed through from Mountaintop. When you answer it you are greeted by Xilin and Hiyamaharu who look very pleased.`
+			`	"We hear that the operation to establish Mooncreek has been successful Captain <last>! How'd things go with you?"`
+			choice
+				`	"Yeah, it was fairly chill."`
+					goto fantastic
+				`	"It was very close out there, there were some losses."`
+			`	Hiyamaharu is inscrutable but Xilin adopts a sombre expression, "Unfortunately there was always a risk of that in this endeavor."`
+			`	"Still," says Hiyamaharu.`
+			label fantastic
+			`	"A great success all round," says Xilin.`
+			`	"The Republic and Hai governments have approved a reward of <payment> by the way, as thanks for your considerable assistance here."`
+			`	"If you have a moment though, we'd appreciate your continued assistance with a bit of PR now. To finalize the station the plan is to facilitate some of the major outfitters to set up on the new station, both from Hai space and from ours. Predominantly this would be so that we can ensure a ready supply of munitions on the station at all times. As well, the long-term garrison to act as security needs to be established. We'd like you to escort each of them on the last leg of their journeys, mostly for the photo ops."`
+			`	"Of course," says Xilin, "we'll understand if you're too busy now."`
```

Comment:
```suggestion
			`	"A great success all round," replies Xilin. "The Republic and Hai governments have approved a reward of <payment> by the way, as thanks for your considerable assistance here.`
			`	"If you have a moment though, we'd appreciate your continued help with a bit of PR now. To finalize the station, the plan is to facilitate some of the major outfitters to set up on the new station, both from Hai space and from ours. Predominantly, this would be so that we can ensure a ready supply of munitions on the station at all times. As well, the long-term garrison to act as security needs to be established. We'd like you to escort each of them on the last leg of their journeys, mostly for the photo ops.`
			`	"Of course," Xilin adds, "we'll understand if you're too busy now."`
```
Paragraph combining and usual edits.